src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -448,12 +448,12 @@ export class InlineChatWidget {
 		if (!viewModel) {
 			return undefined;
 		}
-		for (const item of viewModel.getItems()) {
-			if (isResponseVM(item)) {
-				return viewModel.codeBlockModelCollection.get(viewModel.sessionId, item, codeBlockIndex)?.model;
-			}
+		const items = viewModel.getItems().filter(i => isResponseVM(i));
```

Comment:
we use this to identify which actions should be shown - run/insert/discard or view in chat. formerly, there was only one viewModel item. now, bc of the underlying `ChatWidget`, there are many, so we want to look at the last one.